nautobot/circuits/filters.py

```diff
@@ -60,7 +62,7 @@ class ProviderFilterSet(NautobotFilterSet):
         to_field_name="slug",
         label="Location (slug or ID)",
     )
-    tag = TagFilter()
+    tags = TagFilter()
```

Comment:
Do we get the right behavior by removing this line and instead adding `"tags"` to `Meta.fields`? That was the approach taken in the DCIM filters...

Same comment throughout.

Reply:
I don't think this works period based on testing I'm doing:

```py
# with tag=TagFilter()
>>> f=DeviceTypeFilterSet().filters.get("tag")
>>> f.get_filter_predicate("aqua")
{'tags__slug': 'aqua'}

# with Meta.fields = ["tags"]
>>> f=DeviceTypeFilterSet().filters.get("tags")
>>> f.get_filter_predicate("aqua")
{'tags': 'aqua'}
```
Going to leave it as-is and open an issue to get that fixed in the base filterset.

Reply:
#3154